modules/tribes/client/components/Tribe.js

```diff
@@ -2,22 +2,27 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import classnames from 'classnames';
 import { useTranslation } from 'react-i18next';
+import styled from 'styled-components';
 
 import JoinButton from './JoinButton';
 
 import getTribeBackgroundStyle from './helpers/getTribeBackgroundStyle';
 
+const Container = styled.div`
+  position: relative;
+`;
+
 /**
  * @TODO maybe rename to Tribe
```

Comment:
You did it!

Reply:
Fixed in 9fcdd0ab1e89d508057425b433571a1394c42565